lesson_2_maven/src/main/java/org/example/ValueCalculator.java

```diff
@@ -0,0 +1,66 @@
+package org.example;
+
+public class ValueCalculator {
+
+    public static void main(String[] args) throws InterruptedException {
+        int[] mas = new int[1000000];
+
+        ValueCalculator valueCalculator = new ValueCalculator();
+        long time1 = valueCalculator.consumedTime(mas, 1);
+        long time2 = valueCalculator.consumedTime(mas, 2);
+        System.out.println("Время 1 = " + time1 + "\n" + "Время 2 = " + time2);
```

Comment:
Please, use English comments

Reply:
And also please associate the thread name with a final time calculation

Reply:
Do you mean those variables time1 and 2 or exactly the names of threads?

Reply:
Better, the name of thread